server/container/guice/jpa-common-guice/pom.xml

```diff
@@ -46,6 +46,11 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>james-server-guice-common</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.derby</groupId>
+            <artifactId>derby</artifactId>
+            <scope>test</scope>
```

Comment:
The bad point on this is that the `maven-dependency-plugin` plugin (which is copying the jars in order to have a lib folder for run time) is also copying the tests dependencies ;)